src/__tests__/http-test.js

```diff
@@ -9,20 +9,22 @@
  */
 
 // 80+ char lines are useful in describe/it, so ignore in this file.
+// Also ignore no-unused-expressions rule as chai 'expect' sintax violates it.
```

Comment:
I'm afraid that might be too cryptic for me, do you want these 2 lines to be merged into one (will be looong); or would you flavour  a multi-line comment like so?
```
/*
Line 1
Line 2
*/
```